SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -249,134 +274,124 @@ - (void)connect {
     [self sdl_connect:nil];
 }
 
+/**
+ *  Cleans up after a disconnected accessory by closing any open I/O streams.
+ */
+- (void)disconnect {
+    // Stop event listening here so that even if the transport is disconnected by the proxy we unregister for accessory local notifications
+    [self sdl_stopEventListening];
+    if (self.controlSession != nil) {
+        [self.controlSession stopSession];
+    }
+    if (self.dataSession != nil) {
```

Comment:
Probably don't need to check for `nil` here. If it's `nil` nothing will happen.

Reply:
Fixed. Removed.